plugin/plugin.go

```diff
@@ -216,12 +216,35 @@ func (p *OrmPlugin) getSortedFieldNames(fields map[string]*Field) []string {
 	return keys
 }
 
+func (p *OrmPlugin) generateFields(ormable *OrmableType, message *generator.Descriptor) []string {
+	var fields []string
+
+	for _, v := range message.GetField() {
+		fieldName := generator.CamelCase(*v.Name)
+		fields = append(fields, fieldName)
+	}
+
+	ormFields := p.getSortedFieldNames(ormable.Fields)
+	fieldsJoin := strings.Join(fields, " ")
+	for _, s := range ormFields {
+		if !strings.Contains(fieldsJoin, s) {
```

Comment:
This check doesn't return correct answer if there is e.g. `ProfileId` field in PB message and `Id` in ORM message but not in PB.
Also take in consideration that some fields can conversely exist in PB but not in ORM, this case is not handled in your function.

Reply:
@dkukharau hi, can you send me an example proto file, please? Thank you for the answers! :+1:

Reply:
@serkulesh Do you need any specific example? We have a bunch of them in `example` folder.

Reply:
@dkukharau I think some specific example. I generated file from `example` folder, and compare them with files wich generated by my fork `protoc-gen-gorm`  and don't see differents. If you can send me some one specific proto file, i would be glad. Thank you!

Reply:
Ok.
Try to insert `include: [{type: "int32", name: "id"}]` into `(gorm.opts)` of `TestTypes` from `/example/feature_demo/demo_types.proto` and see that this `id` field will not be generated in `TestTypesORM` when using your fork.
Regarding my second note(some fields can conversely exist in PB but not in ORM), I see that it's not actually true and your code handles correctly this case, but what I expect is that `getOrderedFieldNames` function should only return field names which exist in ORM, so it can be safely used across the plugin. I'm sorry for being inaccurate in my initial review:)

Reply:
@dkukharau your true! My fault, my function has an error in comparing fields. I fixed compare, and created separate file for `getOrderedFieldNames`. Check it please :) Thank you for the example :)